Code/Framework/AzCore/AzCore/Debug/Profiler.h

```diff
@@ -5,310 +5,50 @@
-            LegacyNetworkTrafficReserved,
-            LegacyDeviceReserved,
-
-            // must match EProfiledSubsystem::PROFILE_LAST_SUBSYSTEM
-            LegacyLast,
-
-            // Bulk category via AZ_TRACE_METHOD
-            AzTrace,
-
-            AzCore,
-            AzRender,
-            AzFramework,
-            AzToolsFramework,
-            ScriptCanvas,
-            LegacyTerrain,
-            Terrain,
-            Cloth,
-            // Add new major categories here (and add names to the parallel position in ProfileCategoryNames) - these categories are enabled by default
-
-            FirstDetailedCategory,
-            RendererDetailed = FirstDetailedCategory,
-            ThreeDEngineDetailed,
-            JobManagerDetailed,
-
-            AzRenderDetailed,
-            ClothDetailed,
-            // Add new detailed categories here (and add names to the parallel position in ProfileCategoryNames) -- these categories are disabled by default
-            
-            // Internal reserved categories, not for use with performance events
-            FirstReservedCategory,
-            MemoryReserved = FirstReservedCategory,
-            Global,
-
-            // Must be last
-            Count
-        };
-        static_assert(static_cast<size_t>(ProfileCategory::Count) < (sizeof(ProfileCategoryPrimitiveType) * 8), "The number of profile categories must not exceed the number of bits in ProfileCategoryPrimitiveType");
-
-        /**
-        * Parallel array to ProfileCategory as string category names to be used as Driller category names or for debug purposes
-        */
-        static const char * ProfileCategoryNames[] =
-        {
-            "Any",
-            "Renderer",
-            "3DEngine",
-            "Particle",
-            "AI",
-            "Animation",
-            "Movie",
-            "Entity",
-            "Font",
-            "Network",
-            "Physics",
-            "Script",
-            "ScriptCFunc",
-            "Audio",
-            "Editor",
-            "System",
-            "Action",
-            "Game",
-            "Input",
-            "Sync",
-
-            "LegacyNetworkTrafficReserved",
-            "LegacyDeviceReserved",
-
-            "LegacyLast",
-
-            "AzTrace",
-            "AzCore",
-            "AzRender",
-            "AzFramework",
-            "AzToolsFramework",
-            "ScriptCanvas",
-            "LegacyTerrain",
-            "Terrain",
-            "Cloth",
-
-            "RendererDetailed",
-            "3DEngineDetailed",
-            "JobManagerDetailed",
-            "AzRenderDetailed",
-            "ClothDetailed",
-
-            "MemoryReserved",
-            "Global"
-        };
-        static_assert(AZ_ARRAY_SIZE(ProfileCategoryNames) == static_cast<size_t>(ProfileCategory::Count), "ProfileCategory and ProfileCategoryNames size mismatch");
-    }
-}
+#ifdef USE_PIX
+#include <AzCore/PlatformIncl.h>
```

Comment:
Would it make more sense to have this include outside this #ifdef?

Reply:
At least for now, this is only needed when including pix3.h